README.md

```diff
@@ -111,6 +112,7 @@ Standard TypeScript with React and jsx a11y
         "plugin:matrix-org/typescript",
         "plugin:matrix-org/react",
         "plugin:matrix-org/a11y"
+        "plugin:matrix-org/jest"
```

Comment:
Re: Trying to update the `matrix-js-sdk` usage,

Everytime I `yarn link eslint-plugin-matrix-org` in `matrix-js-sdk`, I see `ESLint couldn't find the config "google" to extend from. Please check that the name of the config is correct.` when running `yarn lint:js` there.

I can't quite work out the `node_modules` resolution problem here. `eslint-config-google` is already installed in `matrix-js-sdk` and should be fine to reference as a peer dependency.

Is there a certain workflow to get this going?

Reply:
:man_shrugging: looks like eslint doesn't work with `yarn link` . Unfortunately I cannot remember what I did when editing this last time :see_no_evil: Maybe just clone it into `node_modules` as a quick fix

Reply:
Still the case. I cloned the repo into `node_modules`, because I don't want to waste my time debugging this + rarely working on it.

Reply:
Just copied the project in and it works 👍 

Thanks for the tip!